test/jdk/java/nio/channels/DatagramChannel/SendReceiveMaxSize.java

```diff
@@ -140,6 +157,9 @@ public void testSendReceiveMaxSize(DatagramChannelSupplier supplier, int capacit
                 if (!Platform.isOSX()) {
                     if (sender.getOption(SO_SNDBUF) < capacity)
                         sender.setOption(SO_SNDBUF, capacity);
+                    if (receiver.getOption(SO_RCVBUF) < capacity+28){
+                        throw new Error("system value " + receiver.getOption(SO_RCVBUF) + " for UDP receive buffer too small to hold capacity " + capacity);
+                    }
```

Comment:
Can you re-format this to avoid this wildly long time, it makes it impossible to look at the changes side-by-side.

Reply:
FWIW - I was concerned that this new check here might make the test fail on other platforms (e.g. Windows/Linux...) I ran the test through our CI and didn't see any new error.

Reply:
The check is a bit dubious as the spec does not guarantee that SO_RCVBUF limits the size of the datagram that can be received. As I read it, you want to check the default value of SO_RCVBUF, in which case it should have after receiver.bind. Also this is a TestNG test so it should be "fail" rather than throwing an error I think.